app/config/settings.py

```diff
@@ -1160,9 +1161,38 @@ def strtobool(val) -> bool:
     "CASES_POST_PROCESSORS", "panimg.post_processors.tiff_to_dzi"
 ).split(",")
 
-# Maximum file size in bytes to be opened by SimpleITK.ReadImage in cases.models.Image.get_sitk_image()
+# Maximum file size in bytes to be opened by SimpleITK.ReadImage in cases.utils.get_sitk_image()
 MAX_SITK_FILE_SIZE = 268_435_456  # 256 mb
 
+SITK_PIXEL_TYPE_TO_BIT_DEPTH = {
```

Comment:
Isn't this already defined in panimg? I'd rather you take this out of settings and just put it in the one place where it is used.